pkg/scheduler/framework/interface_test.go

```diff
@@ -76,6 +89,10 @@ func TestStatus(t *testing.T) {
 				t.Errorf("expect status.IsSuccess() returns %v, but %v", test.expectedIsSuccess, test.status.IsSuccess())
 			}
 
+			if test.status.IsWait() != test.expectedIsWait {
+				t.Errorf("expect status.IsWait() returns %v, but %v", test.expectedIsWait, test.status.IsWait())
```

Comment:
```suggestion
				t.Errorf("status.IsWait() returns %v, but want %v", test.status.IsWait(), test.expectedIsWait)
```

Reply:
we usually word it like:

```
<function name>: want %v, but got %v.
```

Reply:
I don't think we are very consistent with the order, but I rather prefer got before want.

Reply:
Also, this is more in-line with how golang's code style: https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

Reply:
> I don't think we are very consistent with the order, but I rather prefer got before want.

Well, here is some data :)

**want/expect %v but got/get %v**

```
⇒  ag "expect.*but got.*" | wc -l
1442
⇒  ag "want.*but got.*" | wc -l
15
⇒  ag "expect.*but get.*" | wc -l
24
```

**got/get %v but want/expect %v**

```
⇒  ag "got.*but expect.*" | wc -l
19
⇒  ag "got.*but want.*" | wc -l
20
ag "get.*but want.*" | wc -l
1
```

Reply:
correct to got?
If we're going to be consistent with this style, why not just change all the WANTS to GOTS by the way?